guides/micronaut-webjars-stimulus/java/src/main/java/example/micronaut/HomeController.java

```diff
@@ -0,0 +1,17 @@
+package example.micronaut;
```

Comment:
Should all these source files have the license header on them?

Or are we relying on adding the header for the generated code?

At the moment, we seem to have both 🤔

Reply:
I'm going to assume we should have a license here, and I've pushed that change 👍